content/articles/an-introduction-to-defi/index.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /an-introduction-to-defi/
+title: An Introduction to DeFi
+description: This article will introduce the reader on all aspects of DeFi. 
+author: adetu-ridwan
+date: 2022-02-25T00:00:00-21:00
+topics: [Blockchain]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/an-introduction-to-defi/hero.jpeg
+    alt: An Introduction to DeFi Example Image
+---
+The origin of blockchain technology has resulted in a dynamic shift in financial and technological systems. This shift has resulted in how applications are developed and their fundamental structure in their operation.
+<!--more-->
+DeFi referred to as Decentralized Finance,  is a dynamic technological evolution of the traditional financial space. Hence, to adequately define DeFi, we need to have a basic understanding of traditional finance.
+
+Finance (Traditional Finance) is the process that involves the creation, management, and investment of money and financial assets. In traditional finance, all these activities are controlled or regulated by a central authority. 
+
+In traditional finance, the following are the core properties:
+- They hold custody of customers' funds.
+- They serve as an intermediary for transactions
+- They are highly regulated
+- Customers have no privacy to the service provider
+- User data are stored in opaque databases and applications
+
+Decentralized Finance subsequently utilize blockchain technology to solve the centralized authority problem in traditional finance towards decentralizing control and operation.
+
+We can define DeFi by examining the following factors:
+- Custody & Settlement 
+- Censor of Transaction execution
+- Protocol Governance
+
+Decentralized Finance is a system characterized by non-custody settlement, non-censor of transaction execution, and no single protocol governance. 
+
+The other dimension of DeFi is Centralized Decentralized Finance with single protocol governance, while there is a non-custody settlement and non-censor of transactions.
+
+Central Intermediary and Decentralized Finance where there is non-custody of settlement and protocol governance, but there exists the censor of transaction execution is another dimension of Decentralized Finance.
+
+### How DeFi is Built and The Roles in DeFi
```

Comment:
Please capitalize only the first word in a heading. Only capitalize a word if necessary